scripts/select_strains.py

```diff
@@ -2,12 +2,13 @@
 import sys
 import os
 from collections import defaultdict
-from datetime import datetime, timedelta, date
```

Comment:
Small thing, but this would fit better under 14bc83a7972295e4570f4954a8c13ee60df06641 instead of f79489c7e59a2969c7db7da51004b9be8f128a49

Reply:
Thanks, moved to 06f0b8b in latest push.